internal/graph/check_test.go

```diff
@@ -2198,3 +2198,311 @@ func TestCycleDetection(t *testing.T) {
 		})
 	})
 }
+
+func TestGetComputedRelation(t *testing.T) {
+	tests := []struct {
+		name             string
+		model            string
+		objectType       string
+		relation         string
+		expectedError    bool
```

Comment:
**(suggestion): **Could we match against the error message and change this to a string?

Reply:
I will add in a case where the relation is not found.  In this case, the error does not come from our code but from other function.  I don't want to tie the error messages to whatever message from this external function.